src/main/scala/ShowResults.scala

```diff
@@ -0,0 +1,134 @@
+import com.github.tototoshi.csv.CSVWriter
+import scala.concurrent.{Await, ExecutionContext, Future}
+import com.typesafe.scalalogging.LazyLogging
+import scala.collection.JavaConverters._
+import scala.collection.immutable.HashMap
+import scala.collection.mutable.{ArrayBuffer, ListBuffer}
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.io._
+import scala.util._
+import CsvWriter.{writeCountriesWithMaxGasPlants, writeCountriesWithMinGasPlants, writeHeaderForPowerPlants, writePlantsInEachContinent, writeTotalPower, writeYearWithMaxNumOfPlants, outputFilePowerPlants, outputFileGeoStat}
+
+import java.nio.file.{Files, FileSystems}
+
+case class PlantsData(country_code: String, country_long: String, capacity_MV: Double, commissioning_year: String, fuels: Array[String])
+
+case class CountriesAndContinentsData(Continent_Name: String, Three_Letter_Country_Code: String)
+
+object ShowResults {
+
+  val countryCodeInCSV = "country"
+  val countryNameInCSV = "country_long"
+  val capacityInCSV = "capacity_mw"
+  val commissioningYearInCSV = "commissioning_year"
+  val fuelNamesInCSV = Array("fuel1", "fuel2", "fuel3", "fuel4")
+
+  val continentNameInCSV2 = "Continent_Name"
```

Comment:
I dont know what CSV2 and CVS1 files mean. Instead of using numbers in variable names, just define two objects , for example, GeneralStatsFields (with list of fields from CVS1) and <SomeOtherFileFields> for anything related to CVS2.
When you want to use numbers, consider refactoring by extracting several objects/classes to give variables a named scope